modules/front-end/src/app/core/components/env-drawer/env-drawer.component.ts

```diff
@@ -57,10 +57,15 @@ export class EnvDrawerComponent {
   ) {
   }
 
+  ngOnInit(): void {
+    this.envForm.get('name').valueChanges.subscribe((name) => {
+      this.nameChange(name);
+    });
```

Comment:
We should put this logic into `initForm` function.

Reply:
Thank you for your review and valuable feedback! It is a valid point. I will implement the changes.